server/api/operator.go

```diff
@@ -339,6 +340,32 @@ func (h *operatorHandler) Delete(w http.ResponseWriter, r *http.Request) {
 	h.r.JSON(w, http.StatusOK, "The pending operator is canceled.")
 }
 
+// @Tags operator
+// @Summary lists the finished operators in the given time.
+// @Param from query integer false "From Unix timestamp"
+// @Produce json
+// @Success 200 {object} Trend
+// @Failure 400 {string} string "The request is invalid."
+// @Failure 500 {string} string "PD server failed to proceed the request."
+// @Router /operators/record [get]
+func (h *operatorHandler) Record(w http.ResponseWriter, r *http.Request) {
+	var from time.Time
+	if fromStr := r.URL.Query()["from"]; len(fromStr) > 0 {
```

Comment:
It's better to add tests for this field.

Reply:
yes